.github/workflows/integration-tests.yaml

```diff
@@ -16,14 +16,17 @@ on:
       - 'cmd/**'
       - 'charts/**'
       - 'tests/integration/**'
+    tags:
+      - 'v*'
 jobs:
   integration-tests:
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
         cassandra_version: ["3.11.10", "4.0.0"]
-        scenario: ["TestMedusaDeploymentScenario/\"S3\"", "TestMedusaDeploymentScenario/\"Minio\"", "TestReaperDeploymentScenario", "TestMonitoringDeploymentScenario", "TestStargateDeploymentScenario"]
+        #scenario: ["TestMedusaDeploymentScenario/\"S3\"", "TestMedusaDeploymentScenario/\"Minio\"", "TestReaperDeploymentScenario", "TestMonitoringDeploymentScenario", "TestStargateDeploymentScenario"]
```

Comment:
Is this to ensure that the flakey tests (Stargate and Reaper) do not fail the release?

Reply:
No, I'm just trying to get the "Integration Tests" workflow pass as fast as possible to see if the stable release workflow triggers as expected.
So far it's not, but I'll add back all test scenarios before the PR is ready for review.